assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
+    groups = odm.List(odm.Keyword(), default=[], description="**TODO**: **Original**:List of groups related to this scan  **Generated**:A list of group identifiers relevant to this submission, often used for access control.")
+    ignore_cache = odm.Boolean(default=False, description="Bypasses any cached results for services, forcing all services to process the submission anew.")
     ignore_recursion_prevention = odm.Boolean(
-        default=False, description="Should we ignore recursion prevention?")
+        default=False, description="Overrides the default mechanism that prevents recursive scanning of extracted files.")
 
     # TODO: The following three lines can be removed after assemblyline upgrade to 4.6+
     ignore_dynamic_recursion_prevention = odm.Boolean(
-        default=False, description="Should we ignore dynamic recursion prevention?")
-
-    ignore_filtering = odm.Boolean(default=False, description="Should we ignore filtering services?")
-    ignore_size = odm.Boolean(default=False, description="Ignore the file size limits?")
-    never_drop = odm.Boolean(default=False, description="Exempt from being dropped by ingester?")
-    malicious = odm.Boolean(default=False, description="Is the file submitted already known to be malicious?")
-    max_extracted = odm.Integer(default=500, description="Max number of extracted files")
-    max_supplementary = odm.Integer(default=500, description="Max number of supplementary files")
-    priority = odm.Integer(default=1000, description="Priority of the scan")
-    profile = odm.Boolean(default=False, description="Should the submission do extra profiling?")
-    psid = odm.Optional(odm.UUID(), description="Parent submission ID")
-    quota_item = odm.Boolean(default=False, description="Does this submission count against quota?")
-    services = odm.Compound(ServiceSelection, default={}, description="Service selection")
+        default=False, description="Disables dynamic recursion prevention for this submission.")
+
+    ignore_filtering = odm.Boolean(default=False, description="**TODO**: **Original**:Should we ignore filtering services? **Generated**:Indicates if filtering services should be skipped, allowing all files to be processed.")
+    ignore_size = odm.Boolean(default=False, description="Allows the submission to bypass any file size restrictions set by the system.")
+    never_drop = odm.Boolean(default=False, description="**TODO**: **Original**:Exempt from being dropped by ingester? **Generated**:Ensures the submission will not be dropped by the ingestion service, regardless of system load.")
+    malicious = odm.Boolean(default=False, description="**TODO**: **Original**:Is the file submitted already known to be malicious? **Generated**:Flags the submission as known to be malicious, possibly altering its handling.")
```

Comment:
Don't believe it alters the handling of the file if it's known to be malicious, but I believe it does contribute to the voting system by automatically assigning a point in favour of being malicious

Reply:
```suggestion
    malicious = odm.Boolean(default=False, description="Automatically add a malicious vote for the submission")
```

Reply:
Ah, this is good insights!  I propose the following:
```suggestion
    malicious = odm.Boolean(default=False, description="If set to true, the system acknowledges the user's assertion that the file is malicious by automatically casting a malicious vote for the submission.")
```